app/controllers/targets_controller.rb

```diff
@@ -9,12 +9,16 @@ def create
     render json: target, status: :created if target.save!
   end
 
+  def destroy
+    render json: target.destroy
+  end
+
   def index
     render json: current_user.targets
   end
 
-  def destroy
-    render json: target.destroy
+  def update
+    render json: target, status: :accepted if target.update!(target_params)
```

Comment:
What would be the result here if it can't be updated?

Reply:
It'd be this https://github.com/juanmanuelramallo/target-mvd/blob/master/app/controllers/concerns/common_errors_concern.rb#L27